src/Auth.ts

```diff
@@ -720,13 +852,28 @@ export class Auth {
     return tokenStorage.set(JSON.stringify(this.service));
```

Comment:
Let's introduce a new store for available connections, and then here update both stores